### DISCUSSION TITLE
interaction select not working with features property when updating from 6.15.1 to ^7.0.0

### DISCUSSION THREAD

**Discussion by d-koppenhagen** [openlayers/openlayers#14885] [Q&A] [answered]
Hello lovely OpenLayers community!
I am one of the maintainer of the vue3-openlayers project, which wraps Openlayers and provides components for usage with the Vue Framework.

The lib provides a `OlSelectInteraction` component for wrapping `Select`. In most cases this works quite good, but we noticed an issues, that appears, when updating Openlayers from version 6.15.1 to ^7.0.0.

Selecting elements provided via `features` doesn't work anymore.

The issue is described in detail here:
https://github.com/MelihAltintas/vue3-openlayers/issues/175

I read the Changelog from Openlayers but I couldn't find any related entry which helps me to identify the problem.
Is there any idea, people who can help me debugging/identifying the problem?

**Comment by mathiash98** [comment, marked as answer]
See my comment in https://github.com/MelihAltintas/vue3-openlayers/issues/175#issuecomment-1629855104
It seems like `vue3-openlayers` is bundling it's own copy of the `Feature` class. This makes the evaluation of `feature instanceof Feature` return false inside the openlayers codebase.

**Comment by ahocevar** [comment]
That makes sense, @mathiash98. When bundling vue3-openlayers, you need to treat OpenLayers as external, and not include it in the bundle.
